controllers/recruitersController.js

```diff
@@ -0,0 +1,22 @@
+const recruiterQuery = require('../models/recruiters')
+
+/**
```

Comment:
Please remove the word `controller` from the filename
Check this out: https://github.com/Real-Dev-Squad/website-backend/issues/280

Reply:
Done!